signature/docker.go

```diff
@@ -49,16 +50,25 @@ func SignDockerManifest(m []byte, dockerReference string, mech SigningMechanism,
 // VerifyDockerManifestSignature checks that unverifiedSignature uses expectedKeyIdentity to sign unverifiedManifest as expectedDockerReference,
 // using mech.
 func VerifyDockerManifestSignature(unverifiedSignature, unverifiedManifest []byte,
-	expectedDockerReference string, mech SigningMechanism, expectedKeyIdentity string) (*Signature, error) {
+	expectedDockerReference string, mech SigningMechanism, expectedKeyIdentity string) (sig *Signature, err error) {
+	sig, _, err = VerifyDockerManifestSignatureUsingKeyIdentityList(unverifiedSignature, unverifiedManifest, expectedDockerReference, mech, []string{expectedKeyIdentity})
+	return
+}
+
+// VerifyDockerManifestSignatureUsingKeyIdentityList checks that unverifiedSignature uses one of the expectedKeyIdentities
+// to sign unverifiedManifest as expectedDockerReference, using mech.
+func VerifyDockerManifestSignatureUsingKeyIdentityList(unverifiedSignature, unverifiedManifest []byte,
```

Comment:
`VerifyImageManifest`… please, the preferred terminology has shifted since the previous function was added.

Reply:
Should I duplicate/deprecate the existing method signature too for good measure, or are you happy for them to mismatch?

Reply:
I think leaving the old one as is is fine — almost everyone should use `signature.Policy` anyway.